packages/framework/esm-styleguide/src/_overrides.scss

```
@@ -233,6 +233,36 @@
   outline: 2px solid var(--brand-03);
 }
 
+.cds--data-table--xs .cds--overflow-menu {
+  height: spacing.$spacing-07;
+}
+
+.cds--data-table--sm .cds--overflow-menu {
+  height: spacing.$spacing-08;
+}
+
+.cds--data-table--sm .cds--overflow-menu--sm {
+  height: spacing.$spacing-07;
+}
+
+.cds--data-table--xs .cds--overflow-menu {
+  max-height: spacing.$spacing-07;
+}
+
```

```suggestion
.cds--data-table--lg .cds--overflow-menu {
  height: spacing.$spacing-08;
  width: spacing.$spacing-08;
}
```

An example of where this would get used is the `All Encounters` table on the `Visits` page (Note how the overflow menu button gets scrunched when viewing in tablet mode).

---

Thanks a lot for the addition @denniskigen !